packages/plugin-patch/sources/patchUtils.ts

```diff
@@ -138,6 +139,17 @@ export async function loadPatchFiles(parentLocator: Locator | null, patchPaths:
       },
     }, patchPath)));
   });
+
+  // Normalizes the line endings to prevent mismatches when cloning a
+  // repository on Windows systems (the default settings for Git are to
+  // convert newlines back and forth, which would mess with the checksum)
+  return patchFiles.map(definition => {
+    if (typeof definition === `string`) {
+      return definition.replace(/\r\n?/g, `\n`);
```

Comment:
```suggestion
      return definition.replace(/\r?\n/g, `\n`);
```

Reply:
I think the existing regex is correct - the intent is to replace `\r` and `\r\n` by `\n`

Reply:
Yeah, it is... my suggestion would also match `\n` and replacing `\n` with `\n` is pointless